Livecheckables/djvulibre.rb

```diff
@@ -0,0 +1,4 @@
+class Djvulibre
+  livecheck :url   => "https://git.code.sf.net/p/djvu/djvulibre-git.git",
+            :regex => /^release.(\d+(?:\.\d+)+)$/
```

Comment:
```suggestion
            :regex => /^release.?(\d+(?:\.\d+)+)$/
```

Note that `.` in regexes is a wildcard and means "match anything". To match a period, you have to escape it like `\.`. However, in this case, there's always a chance that the format could change from `release.3.5.27` to something like `release-3.5.27`, `release_3.5.27`, or maybe even `release3.5.27`, so I'm okay with using a single wildcard here. We avoid wildcards that match more than one character (e.g., `.*`, `.+`, `.*?`, `.+?`).

I've simply added a `?` here to make it optional, to account for any future releases with typos.

Reply:
Works for me. Honestly, I dont think we need to be too accommodate with the typo change though.

Reply:
You may be surprised how often things like this happen and break livecheckables. I'm saying this having seen it firsthand in livecheckables several times now. In this case, we lose nothing by being ever-so-slightly more permissive but we gain a bit by being a little less brittle.